### PR TITLE
Updating vLLM tests to handle changes between vLLM version 0.2.3 and 0.3.0

### PR DESCRIPTION
**Pull request by oandreeva-nv**
Updated accuracy test not to run 2 vLLM engines at the same time, due to https://github.com/vllm-project/vllm/issues/2248

### CODE REVIEW

File: ci/L0_backend_vllm/accuracy_test/test.sh [hunk 1]
```diff
@@ -44,6 +45,13 @@ sed -i 's/"gpu_memory_utilization": 0.5/"gpu_memory_utilization": 0.3/' models/v
 
 RET=0
 
+set +e
+# Need to generate baseline first, since running 2 vLLM engines causes
+# memory issues: https://github.com/vllm-project/vllm/issues/2248
+python -c "import accuracy_test; accuracy_test.prepare_vllm_engine_outputs()" >> $VLLM_ENGINE_LOG 2>&1 & BASELINE_PID=$!
```
Comment (jbkyang-nvi) [inline ci/L0_backend_vllm/accuracy_test/test.sh:51]: Nit: I think it's a bit confusing to have us call the script `accuracy_test.prepare_vllm_engine_outputs()` and then use `CLIENT_PY`. Can we generalize a bit here?

Something like 
```
python3 $CLIENT_PY --generate-baseline
```
and argparse on the script?

Comment (oandreeva-nv) [reply ci/L0_backend_vllm/accuracy_test/test.sh:51]: Nice idea, thanks! Will incorporate

Comment (oandreeva-nv) [reply ci/L0_backend_vllm/accuracy_test/test.sh]: revised

**Review by jbkyang-nvi — APPROVED** [review]
lgtm!

**Review by jbkyang-nvi — APPROVED** [review]

**Review by jbkyang-nvi — APPROVED** [review]
Thanks!

**Review by pskiran1 — APPROVED** [review]

**Pull request merged**